pandas/tests/io/test_sql.py

```diff
@@ -712,11 +947,31 @@ def sqlite_buildin_types(sqlite_buildin, types_data):
     mysql_connectable_types + postgresql_connectable_types + sqlite_connectable_types
 )
 
-all_connectable = sqlalchemy_connectable + ["sqlite_buildin"]
+adbc_connectable = [
+    pytest.param("postgresql_adbc_conn", marks=pytest.mark.db),
+    pytest.param("sqlite_adbc_conn", marks=pytest.mark.db),
```

Comment:
These shouldn't need the `pytest.mark.db` mark right since it uses a local sqllite file?